stage/dotnet/dotnet-sdk/cli/cli.csproj

```diff
@@ -9,7 +9,7 @@
   <RestoreSources>$(RestoreSources);../src/bin/Debug;https://api.nuget.org/v3/index.json</RestoreSources>
 </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="7.5.1" />
+    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="8.0.1" />
```

Comment:
I wonder why these updates aren't covered by Dependabot: https://github.com/octokit/source-generator/blob/main/.github/dependabot.yml#L3-L19

This shouldn't be something we have to catch manually...it's a frustrating experience to clone the repo and not be able to run the CLI because of issues like this.

Reply:
Yeah I agree.. I was planning on digging in once I wrapped up things here.